python/drned_xmnr/op/config_op.py

```diff
@@ -68,6 +68,24 @@ def perform(self):
                 str([self.state_filename_to_name(st) for st in state_files])}
 
 
+class ViewStateOp(ConfigOp):
+    action_name = 'view state'
+
+    def _init_params(self, params):
+        self.state_name = params.state_name
+
+    def perform(self):
+        self.log.debug("config_view_state() with device {0}".format(self.dev_name))
+        state_name = self.state_name
+        state_filename = self.state_name_to_filename(state_name)
+        if state_filename == []:
+            raise ActionError('no such state: ' + self.state_name)
```

Comment:
This check is redundant. First - for every state name there should always be a state filename (except perhaps in pretty extreme scenarios), the state name is modeled as a leafref to state name list, and the names are generated based on existing state files. Second - `state_name_to_filename` is basically just a string conversion, so even if the file did not exist, this function would still return nonempty string.